Remove deprecations in app system guides; use manifest-2.0.xml in exa… (#857)
shopware/docs: guides/plugins/apps/administration/add-custom-modules.md

To configure your module you can set it up with with some additional attributes.

* `name` \(required\): The technical name of the module. This is the name your module is referenced with.

> * `parent` \(required\): The Administration navigation id from the menu item that serves as the parent menu item. If omitted your module will be listed under the "My apps" menu entry.

Reviewer: ```suggestion
* `parent` \(required\): The Administration navigation id of the menu item that serves as the parent menu item.
```

It can't be omitted anymore 😄
